psydac/linalg/tests/test_linalg.py

```diff
@@ -0,0 +1,959 @@
+import pytest
+import numpy as np
+
+from psydac.linalg.block import BlockLinearOperator, BlockVector, BlockVectorSpace
+from psydac.linalg.basic import ZeroOperator, IdentityOperator, ComposedLinearOperator, SumLinearOperator, PowerLinearOperator, InverseLinearOperator, ScaledLinearOperator
+from psydac.linalg.stencil import StencilVectorSpace, StencilVector, StencilMatrix
+from psydac.linalg.solvers import BiConjugateGradient, ConjugateGradient, PConjugateGradient, inverse
+from psydac.ddm.cart         import DomainDecomposition, CartDecomposition
+#===============================================================================
+
+# 11.12.22: Check for redundancy(?) in pytests 1,2 and 4, document 1,2
+
+n1array = [2, 7]
+n2array = [2, 3]
+p1array = [1, 3]
+p2array = [1, 3]
+
+def is_pos_def(x):
+    assert np.all(np.linalg.eigvals(x) > 0)
+
+def l2_norm_diff(x,y):
+    l = len(x)
+    assert l == len(y)
+    vec = [(x[i]-y[i])**2 for i in range(l)]
+    norm = np.sqrt(sum(vec))
+    return norm
```

Comment:
I think that you should use `numpy.linalg.norm(x - y, ord=2)`, which is surely much faster than this function.

Reply:
:') this function was not meant to be seen by anybody, please forget it ever existed